ferrocene/doc/internal-procedures/src/subtree-pulls.rst

```diff
@@ -35,6 +35,21 @@ request automatically, only requiring sign off of a reviewer to get merged.
 Sometimes merge conflicts may occur in which case automation opens an issue
 describing the conflict and requiring the upstream pull to be done manually.
 
+Backtrace subtree pull
+^^^^^^^^^^^^^^^^^^^^^^
+
+When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
```

Comment:
rST `<code>` markup is done with double backtick, not single

Reply:
alright, needs fixing in a future pr then

Reply:
Probably for the whole Ferrocene docs 👍🏾